scripts/package/package.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const bcd = require('./package-bcd');
+const examples = require('./package-examples');
+const attributes = require('./package-attributes');
+const prose = require('./package-prose');
+const contributors = require('./package-contributors');
+
+const htmlElements = '/content/html/elements';
+
+const writeToFile = (propertyName, json) => {
+  const data = {
+    html: {
+      elements: {
+        [propertyName]: json,
+      }
+    }
+  };
+
+  const dest = path.join(process.cwd(),'packaged', `${propertyName}.json`);
+  const destDir = path.dirname(dest);
+  if (!fs.existsSync(destDir)) {
+      fs.mkdirSync(destDir, { recursive: true });
+  }
+  fs.writeFileSync(dest, `${JSON.stringify(data, null, 2)}`);
+};
+
+function package(elementName) {
+    const elementPath = path.join(process.cwd(), htmlElements, elementName);
+
+    if (!fs.exists(elementPath)) {
```

Comment:
Or, better, leave it to be async exists and use promises. E.g.

```javascript

async function package(elementName) {
   const elementPath = path.join(process.cwd(), htmlElements, elementName);
   const exists = await fs.exists(elementPath);
    if (!exists) { 
      ...
```

It would be a shame if this package becomes synchronous. With all the disk IO my gut's telling me it would be faster that way.

Reply:
I don't think this blocks https://github.com/mdn/sprints/issues/1499, but agree we should do it, so I have filed https://github.com/mdn/stumptown-experiment/issues/20